### PR TITLE
fix: regexp for command validation

### PR DESCRIPTION
**Pull request by xhyrom**
**Please describe the changes this PR makes and why it should be merged:**

**Status and versioning classification:**

### CODE REVIEW

**Comment by xHyroM** [comment]
@garlic-team pack

**Comment by xHyroM** [comment]
@garlic-team pack

**Comment by xhyrom** [comment]
@garlic-team pack

**Comment by xhyrom** [comment]
@S222em fast review please :D

File: src/lib/util/regexes.ts [hunk 1]
```diff
@@ -32,4 +32,4 @@ export const mentionableRegexp = /^(?:<@!?)?(?:<@&?)?([0-9]+)>?$/;
  * Regex for command and option names
  * @raw `/^[\P{Lu}\p{N}_-]+$/u`
  */
-export const commandAndOptionNameRegexp = /^[\P{Lu}\p{N}_-]+$/u;
+export const commandAndOptionNameRegexp = /^[\P{Lu}\p{N}_-aA-zZ]+|[\s]$/u;
```
Comment (S222em) [inline src/lib/util/regexes.ts]: This regex can be simplified further (I think atleast)

**Review by S222em — CHANGES_REQUESTED** [review]

File: src/lib/util/regexes.ts [hunk 2]
```diff
@@ -30,6 +30,6 @@ export const mentionableRegexp = /^(?:<@!?)?(?:<@&?)?([0-9]+)>?$/;
 
 /**
  * Regex for command and option names
- * @raw `/^[\P{Lu}\p{N}_-]+$/u`
+ * @raw `/^[a-zA-Z1-9]/`
  */
-export const commandAndOptionNameRegexp = /^[\P{Lu}\p{N}_-]+$/u;
+export const commandAndOptionNameRegexp = /^[a-zA-Z1-9]/;
```
Comment (S222em) [inline src/lib/util/regexes.ts]: What about spaces?

**Review by S222em — CHANGES_REQUESTED** [review]

File: src/lib/util/regexes.ts (hunk 2 above)
Comment (xhyrom) [reply src/lib/util/regexes.ts]: ah, im idiot

File: src/lib/util/regexes.ts [hunk 3]
```diff
@@ -30,6 +30,6 @@ export const mentionableRegexp = /^(?:<@!?)?(?:<@&?)?([0-9]+)>?$/;
 
 /**
  * Regex for command and option names
- * @raw `/^[\P{Lu}\p{N}_-]+$/u`
+ * @raw `/^[a-zA-Z1-9]/`
  */
-export const commandAndOptionNameRegexp = /^[\P{Lu}\p{N}_-]+$/u;
+export const commandAndOptionNameRegexp = /[a-zA-Z1-9-\s]/;
```
Comment (S222em) [inline src/lib/util/regexes.ts]: I'm not sure if the `-` is needed before the `\s`, I am not a regex god so I wouldn't know. If it works its fine

**Review by S222em — APPROVED** [review]

**Pull request merged**